gshoppingflux/gshoppingflux.php

```diff
@@ -1634,7 +1634,7 @@ public function renderInfo()
 		));
 	}
 
-	public function customGetNestedCategories($shop_id, $root_category = null, $id_lang = false, $active = true, $groups = null, $use_shop_restriction = true, $sql_filter = '', $sql_sort = '', $sql_limit = '')
```

Comment:
Yes, I agree with you! As I didn't understand the purpose of this variable I removed it :)